ci-operator/step-registry/assisted/capi/version-discovery/assisted-capi-version-discovery-commands.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```

Comment:
I think probably this file should live in the project: if we want to execute it manually, we easily could. Then we can call it from CI with the necessary env vars